php/libraries/NDB_BVL_InstrumentStatus.class.inc

```diff
@@ -160,12 +160,12 @@ class NDB_BVL_InstrumentStatus
     /**
      * Sets the current data entry status of the instrument
      *
-     * @param string $status the new status from the set (NULL,
-     *                       'In Progress','Not Complete','Complete')
+     * @param ?string $status the new status from the set (NULL,
+     *                        'In Progress','Not Complete','Complete')
      *
      * @return void
      */
-    function setDataEntryStatus(string $status): void
+    function setDataEntryStatus(?string $status): void
```

Comment:
Why is this changing? This adds an ambiguity between the empty string and NULL. (Same with the others modified in this file.)

Reply:
This is so that `NULL`can be accepted as a parameter. It says in the description that the parameter `$status` should accept `NULL`. That being said, the values are `NULL` when the instrument is created, but I'm not sure why the status would ever go back to `NULL`.